src/ngraph/serializer.cpp

```diff
@@ -2676,10 +2685,18 @@ json JSONSerializer::serialize_node(const Node& n)
     }
     case OP_TYPEID::Pad:
     {
-        auto tmp = dynamic_cast<const op::Pad*>(&n);
-        node["padding_below"] = tmp->get_padding_below();
-        node["padding_above"] = tmp->get_padding_above();
-        node["pad_mode"] = tmp->get_pad_mode();
+        if (op_version == 0)
+        {
+            auto tmp = dynamic_cast<const op::Pad*>(&n);
```

Comment:
```suggestion
            auto tmp = dynamic_cast<const op::v0::Pad*>(&n);
```

(Best to be explicit when dealing with "version-aware" code, IMHO)